ingestion/src/metadata/ingestion/bulksink/metadata_usage.py

```diff
@@ -102,7 +102,7 @@ def __populate_table_usage_map(
             self.table_usage_map[table_entity.id.__root__] = {
                 "table_entity": table_entity,
                 "usage_count": table_usage.count,
-                "sql_queries": table_usage.sqlQueries,
+                "sql_queries": list(table_usage.sqlQueries),
```

Comment:
Why are we doing this if the `sqlQueries` is already a List? 
```python
    sqlQueries: Optional[List[table.SqlQuery]] = Field(
        None, description='List of SQL Queries associated with table'
    )
```

Reply:
if you notice [here](https://github.com/open-metadata/OpenMetadata/blob/4885bc4a9ab78ca076eab549ce3db0769ad24261/ingestion/src/metadata/ingestion/bulksink/metadata_usage.py#L215) we are passing the same object of table_usage for each table_entity, and in the map both the table entities will have the same reference to the list of `sqlQueries` and when we are extending the list extend happens for all table entities which should not be the case 

just to make it more clear, below code snippet can summarise the scenario   
![image](https://user-images.githubusercontent.com/39544459/214752548-ca238417-e77a-4683-9134-103c73b8b92d.png)


hence instead of linking the reference to the list, I'm creating a new list and linking that to map, and yes I should take care of nulls here

Reply:
Oh, I see. It makes total sense! Thank you for the explanation :)

Reply:
let's check for nulls and bring it in. Thanks both